apps/explorer/src/app/app.tsx

```diff
@@ -40,6 +41,15 @@ function App() {
   return (
     <TendermintWebsocketProvider>
       <NetworkLoader cache={cacheConfig}>
+        <AnnouncementBanner>
+          <div className="font-alpha calt uppercase text-center text-lg text-white">
+            <span className="pr-4">The Mainnet sims are live!</span>
+            <ExternalLink href="https://fairground.wtf/">
```

Comment:
(non-blocking): I thought the link was supposed to be this one `https://fairground.wtf/sim1` but perhaps is not generic. and that one is only for #1